hachoir/parser/misc/fit.py

```diff
@@ -0,0 +1,170 @@
+"""
+Garmin fit file Format parser.
+
+Author: Sebastien Ponce <sebastien.ponce@cern.ch>
+"""
+
+from hachoir.parser import Parser
+from hachoir.field import FieldSet, Int8, UInt8, Int16, UInt16, Int32, UInt32, Int64, UInt64, RawBytes, Bit, Bits, Bytes, String, Float32, Float64
+from hachoir.core.endian import BIG_ENDIAN, LITTLE_ENDIAN
+
+field_types = {
```

Comment:
is this list sorted somehow? maybe sort by the key, or group integer types together? it's just a remark.

Reply:
To be honest, it's the order I found in the garmin SDK, and I'm now wondering indeed whether there is any logic... I believe yes though : growing size and for each size unsigned and signed version. Now there are some duplicas as bool for example ends up as a UInt8. I'll add comments on the line of the actual type to make it clearer.